README.md

```diff
@@ -13,3 +13,23 @@ The *tutorials* folder contains walkthroughs which demonstrate basic Certifai fu
 ## Certifai Documentation
 
 Refer to the Cortex Certifai documentation: https://cognitivescale.github.io/cortex-certifai/docs/about for detailed information about Cortex Certifai.
+
+## Notebooks Table of Contents
+
+| Notebook Name | Description | Task Type | Model Types | Evaluation Types | Key Properties |
+| --- | --- | --- | --- | --- | --- |
+| [AnalyzeFeatureUsage.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/analyze_feature_usage) - Analyzing Feature Usage | Illustrates the use of Certifai to create and run an explanations scan of a locally defined model from first principles. It then analyzes and displays feature distribution of the counterfactuals.  |  Binary classification | Decision Tree <br /> Logistic Regression | Explanations |  Feature Occurrence <br /> Frequency by Model |
+| [BringInYourOwnModel.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/tutorials/bringing_in_your_own_model/part_one) - Building a scan programmatically with your own model | Part one of a tutorial that takes data scientists through model preparation and scan. This notebook/tutorial is also packaged as part of the toolkit and can be accessed at `examples/notebooks/BringingInYourOwnModel.ipynb`. | Binary Classification | SVM <br /> Logistic Regression | Fairness  <br /> Explainability <br />  Robustness |   |
+| (certifai_multiclass_evaluation)[model_train_part1.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates how to start the multiclass classification task type scan. | Multiclass Classification | Logistic Regression | Performance Metric: Accuracy  <br /> Explainability-Explanations <br /> Fairness (by feature) | One-hot encoding <br /> Target encoded features |  
+| [certifai_multiclass_evaluation_part2.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates the second part of the multiclass classification task type. | Multiclass Classification | Logistic Regression | Performance Metric: Accuracy <br /> Explainability-Explanations <br /> Fairness (by feature) | One-hot encoding <br /> Target encoded features |
```

Comment:
Probably shouldnt list these notebooks individually because they really are just breaking up what is one overall example. Point to the folder and say notebooks  in it illustrate how to handle a multi-class example where some of the data is target encoded.  There's an update to this example coming which will add even more notebooks (and will show how to scan in a notebook and then transition to using Pro in Azure) - but I guess we can cross that bridge when its built!

Reply:
I second what Christine said above.

I also think that under evaluation types, this should say: "Performance (Accuracy)", "Performance (Metric: Accuracy)", or maybe even just "Performance".  My thinking is that we just want to be clear that "Performance" is the evaluation type